src/script/backport-create-issue

```diff
@@ -305,20 +305,37 @@ def mark_as_processed(r, issue):
     This script will add a custom Tag to indicate whether the tracker was
     already processed for backport tracker creation.
     """
-    custom_fields = list(issue['custom_fields'].values())
-    for i, field in enumerate(custom_fields):
-        if field['name'] == 'Tags':
-            if tag_backport_processed not in field['value']:
-                if field['value']:
-                    custom_fields[i]['value'] += (tag_separator +
-                                                  tag_backport_processed)
-                else:
-                    custom_fields[i]['value'] = tag_backport_processed
-                logging.info("%s adding tag '%s'", url(issue),
-                             tag_backport_processed)
-                r.issue.update(issue.id, custom_fields=custom_fields)
-                return
 
+    logging.debug("custom_fields: %s", list(issue['custom_fields']))
+
+    try:
+        tags_cf = next(filter(lambda x: x['id'] == CF_TAGS, issue['custom_fields']))
+    except StopIteration:
+        tags_cf = {
+          'id': CF_TAGS,
+          'value': '',
+        }
```

Comment:
For a fallback assignment I find it easier to use `next(iter, default)`:

```suggestion
    tags_cf = next(
        filter(lambda x: x['id'] == CF_TAGS, issue['custom_fields']),
        default=dict(id=CF_TAGS, value='')
    )
```

Reply:
Good suggestion, thanks.